projects/novo-elements/src/elements/date-picker/DatePickerInput.ts

```diff
@@ -89,15 +89,15 @@ export class NovoDatePickerInputElement implements OnInit, ControlValueAccessor
     private _changeDetectorRef: ChangeDetectorRef,
     public dateFormatService: DateFormatService,
   ) {
-    this.placeholder = this.labels.dateFormatPlaceholder;
+    this.placeholder = this.labels.dateFormatString().toUpperCase() || this.labels.dateFormatPlaceholder;
   }
 
   ngOnInit() {
     this.userDefinedFormat = this.format ? !this.format.match(/^(DD\/MM\/YYYY|MM\/DD\/YYYY)$/g) : false;
     if (!this.userDefinedFormat && this.textMaskEnabled && !this.allowInvalidDate) {
       this.maskOptions = this.maskOptions || {
-        mask: [/\d/, /\d/, '/', /\d/, /\d/, '/', /\d/, /\d/, /\d/, /\d/],
```

Comment:
so the getDateMask() mask looks like a more flexible version of this mask bc it accounts for slashes, dots, & dashes?

Reply:
Yes, precisely. It allows the use of multiple different date formats from different locales such as: YYYY/MM/DD, DD.MM.YYYY etc.